types/react-flag-icon-css/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for react-flag-icon-css 1.0
+// Project: https://github.com/matteocng/react-flag-icon-css#readme
+// Definitions by: Jon Freedman <https://github.com/jonfreedman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import { PureComponent } from 'react';
+
+export type Size = 'lg' | '2x' | '3x' | '4x' | '5x';
+
+export interface Options {
```

Comment:
Can you name this as FlagIconOptions instead to make it clear ?

Reply:
done